client/.storybook/theme.js

```diff
@@ -0,0 +1,8 @@
+import { create } from '@storybook/theming';
+
+export default create({
+  base: 'light',
+  brandTitle: 'freeCodeCamp.org',
+  brandImage:
+    'https://cdn.freecodecamp.org/platform/universal/fcc_secondary.svg'
+});
```

Comment:
I browsed the cdn repo but couldn't find a logo file that has both text and gray90 background color. I resorted to the secondary logo as the text of the primary one is not readable on Storybook's default sidebar background.

Reply:
We have png assets with set backgrounds on the design style guide. https://github.com/freeCodeCamp/design-style-guide/blob/master/downloads/fcc_primary_large.png but they are not on the CDN.

We could default to the gray85 for the background and use the primary SVG down the road.

Reply:
I just tried it out and the PNG doesn't seem to have a background either

<img width="894" alt="Screen Shot 2021-05-11 at 14 05 12" src="https://user-images.githubusercontent.com/25715018/117772985-f9f3ac00-b261-11eb-9af0-2e5f3ff31f3f.png">

---
Storybook has very limited [theming options](https://github.com/storybookjs/storybook/blob/a3d63d6a3fe28b6e3577ba9e50370a0d14ceca7e/lib/theming/src/types.ts#L5). In order to have a dark background for the primary logo, I would need to set `gray85` to the entire sidebar and change the text color; but it affects the Controls panel, which doesn't have dedicated styling options.

So I think we would still need an image file with a set background on CDN if we want to use the primary logo. 

<img width="1680" alt="Screen Shot 2021-05-11 at 13 50 49" src="https://user-images.githubusercontent.com/25715018/117773455-869e6a00-b262-11eb-93d7-4f11547c3a2e.png">